source/sync/data-model/data-model-map.txt

```diff
@@ -236,14 +240,49 @@ refer to :ref:`BSON Types - Array <schema-type-array>`.
 App Services Schemas support certain constraints that Realm Object Schemas do not,
 such as specifying the minimum and maximum number of items.
 
+.. _mixed-properties:
+
+Mixed Properties
+~~~~~~~~~~~~~~~~
+
+Both Realm Object Schemas and App Services Schemas support mixed
+type properties.
+
+A mixed field can contain any supported data type and 
+functionally acts as an object or document without a pre-defined
+structure. 
+
+For more information on modeling mixed properties in 
+an App Services Schema, refer to :ref:`Mixed - Schema Types <schema-type-mixed>`.
+
+Collections in Mixed Properties
+```````````````````````````````
+
+.. include:: /includes/note-collections-in-mixed-sync.rst
+
+You can leverage collections (arrrays or dictionaries) in mixed properties to store data that does not fit into a
+pre-defined schema, such as variable JSON data or complex MongoDB documents. Because collections of mixed data can contain other collections of
+mixed data, you can nest data into complex structures. 
+For an example, refer to the :ref:`nested-collections-data-model-example` on this page.
+
+Currently, App Services Schemas and Realm Object Schemas in the following SDKs 
+support collections in mixed data types:
+
+- Flutter SDK v2.0.0 or later: :ref:`RealmValue - Flutter SDK
+  <flutter-realm-value>`
+
+Other SDKs will support collections in mixed data types in future releases.
+
+.. TODO: Add links to other SDKs when available.
```

Comment:
[q] will this TODO be hidden or is this temporary?

Reply:
it's only visible in the .txt file (i.e. it doesn't display on the page)